SPECS/openssh/openssh.spec

```diff
@@ -32,6 +32,9 @@ Patch306:       pam_ssh_agent_auth-0.10.2-compat.patch
 # Fix NULL dereference from getpwuid() return value
 # https://sourceforge.net/p/pamsshagentauth/bugs/22/
 Patch307:       pam_ssh_agent_auth-0.10.2-dereference.patch
+#CVE Patches
```

Comment:
Nit: Add a space after the # to match other comments

Reply:
won't fix.